composer.json

```diff
@@ -23,14 +23,14 @@
         }
     ],
     "require":     {
-        "php": "^5.4|^7",
+        "php": ">=5.5.9",
```

Comment:
`^5.5.9 || ^7.0 || ^8.0`

Reply:
So, this was already discussed above... In conclusion, what should this be set to?

Reply:
Well, we know Symfony takes the view that no upper bound is best practice, but Laravel and Compose take the opposite view, that there should always be upper bounds (https://github.com/composer/composer/blob/master/composer.json#L25, https://github.com/laravel/framework/blob/7.x/composer.json#L18).

Reply:
The fact that composer itself thinks upper bounds are important matter more than Symfony's view IMO.

Reply:
Especially after composer added `--ignore-platform-dep=php` as a weaker alternative to `--ignore-platform-deps`, Symfony's arguments are weak.

Reply:
I explained the arguments for using `>=` in https://github.com/symfony/symfony/pull/36876, they're strong. I'm sorry they're apparently hard to understand.

Reply:
Agree to disagree. :P